telemetry/definitions/commonDefinitions.json

```diff
@@ -3890,8 +3892,8 @@
                     "required": true
                 },
                 {
-                    "type": "reason",
-                    "required": false
+                    "type": "codeTransformRuntimeError",
+                    "required": true
```

Comment:
Required here because when `mvn` command fails we should always have an error message to emit.

Reply:
Same as my other comment above

Reply:
If IntelliJ uses the `codeTransformRuntimeError` field already then we should use this field. If not, we should move the error into the `reason` field if possible.

Reply:
What kind of error message is expected here? Do you have any sample format. 
Telemetry metrics are for overall analysis and we don't want to have long texts. maybe a gist in reason is good practice. 

if you need long error messages, we can dump it to cloudwatch logs